rdm/init_files/Makefile

```diff
@@ -17,12 +17,12 @@ pdfs: $(RELEASE_PDFS)
 jsons: $(RELEASE_JSONS)
 
 .PHONY:
-github_issues: $(SYSTEM_CONFIG)
+github_issues: $(CONFIG)
```

Comment:
Perhaps a better name for this make target would be `history`?

Reply:
I actually changed the recipe a bit so that this would create `data/history.yml`, which I believe is appropriate here. I also moved the recipe down to the bottom since it is "PHONY" after all :)